app/scripts/dateformat.js

```diff
@@ -0,0 +1,226 @@
+/*
```

Comment:
So this turned out to be a little more complicated than David thought in the meeting.  To do a bower install you need to find the correct name of the bower package.  A little searching on http://bower.io/search/ helped me find that the "date.format" package is the one you want.  So the following command should work:
`bower install date.format --save`
Please let me know if there's anything else I can do to help!